templates/an-cash-dao/scripts/deploy.js

```diff
@@ -0,0 +1,81 @@
+const { utf8ToHex } = require('web3-utils')
+const { pct16, bn, ONE_DAY } = require('@aragon/contract-helpers-test')
+
+const Template = artifacts.require('ANCashDAOTemplate')
+
+const DEFAULTS = {
+  token: {
+    name: 'AN Cash DAO Token',
+    symbol: 'ANCDT',
+    holders: ['0x0090aED150056316E37FE6DFa10Dc63E79D173B6'],
+    stakes: [1]
+  },
+  agreement: {
+    title:        'Aragon Network Cash Agreement',
+    content:      utf8ToHex('ipfs:QmPvfWUNt3WrZ7uaB1ZwEmec3Zr1ABL9CncSDfQypWkmnp'),
+  },
+  disputableVoting: {
+    duration:               ONE_DAY * 5,
+    support:                pct16(50),
+    minQuorum:              pct16(50),
+    executionDelay:         0,
+    delegatedVotingPeriod:  ONE_DAY * 2,
+    quietEndingPeriod:      ONE_DAY,
+    quietEndingExtension:   ONE_DAY / 2,
+    actionCollateral:       bn(0),
+    challengeCollateral:    bn(0),
+    challengeDuration:      ONE_DAY * 3,
+  },
+}
+
+const CONFIG = {
+  rinkeby: {
+    ...DEFAULTS,
+    ens:            '0x98Df287B6C145399Aaa709692c8D308357bC085D',
+    registrar:      '0x3665e7bfd4d3254ae7796779800f5b603c43c60d',
+    daoFactory:     '0xad4d106b43b480faa3ef7f98464ffc27fc1faa96',
```

Comment:
```suggestion
    daoFactory:     '0x89d87269527495ac29648376d4154ba55c4872fc',
```

This one (v0.8.1) has ACLv1.1 for ACL Oracles.